src/Components/ServiceScene/Breakdowns/FieldsBreakdownScene.tsx

```diff
@@ -463,7 +353,14 @@ export class FieldsBreakdownScene extends SceneObjectBase<FieldsBreakdownSceneSt
               />
             )}
           </div>
-          <div className={styles.content}>{body && <body.Component model={body} />}</div>
+
+          {/* @todo why are the types like this? */}
```

Comment:
We should know 😆

Reply:
This helped: https://www.executeprogram.com/courses/advanced-typescript/lessons/type-algebra, basically some of these objects had different definitions (at least one of the methods had differing access control modifiers) of sceneObject methods, so when attempting to define a type that is the union of the various body types, typescript reduced the definition to never. To get around this I made the definition of body a bit more verbose, and defined each union as the intersection of the object and SceneObject. Which appears to work as expected, and is probably better then a type assertion!

Might be nice to come back to this when we have time and update scenes core to sync the access control modifiers